.github/workflows/lint_go-vet.yml

```diff
@@ -0,0 +1,24 @@
+name: lint_go-vet
```

Comment:
Rename to `govet-lint`

Reply:
Actually, I prefer to have `lint_` as a prefix and the name of the linter as the suffix. This makes it easier to spot what the job is doing.